docs/ref/forms/fields.txt

```diff
@@ -497,7 +497,7 @@ For each field, we describe the default widget used if you don't specify
 
     .. attribute:: input_formats
 
-        A list of formats used to attempt to convert a string to a valid
+        A generator of formats used to attempt to convert a string to a valid
```

Comment:
`iterable` is more accurate:
```suggestion
        An iterable of formats used to attempt to convert a string to a valid
```

Reply:
@stefan6419846 Does it work for you?

Reply:
Yes, `iterable` should work here and make it more obvious the we should not expect a list to be returned.

Thinking about this, should the other field docs for `DateField` and `TimeField` updated as well to just state `iterable` instead of `list` to use common wording, or is there enough benefit of using different terms here as the underlying types actually differ?

Reply:
> Thinking about this, should the other field docs for `DateField` and `TimeField` updated as well to just state `iterable` instead of `list` to use common wording, or is there enough benefit of using different terms here as the underlying types actually differ?

Using `iterable` for all `input_formats` seems reasonable.